docs/index.md

```diff
@@ -1,14 +1,14 @@
 # Welcome to Ultra
 
-Ultra is a platform that aims at providing Flipkart customers with shopping experiences on categories that Flipkart does not handle. It is an idea designed to pave new ways for users to interact with any external application within Flipkart with security and trust. With Flipkart having one of the largest user base in the country, building a distribution platform to host experiences or services (for example, booking tickets, bill payments) offered by other non-competing players (such as, MakeMyTrip, Phone Pe and so on) can help customers get access to a single shopping destination and helping developers get wider access while Flipkart gets a greater share of the user’s wallet.
+Ultra is an application development platform within Flipkart application offering an open window for external companies or organizations that are existing or may want to become Flipkart business partners. Ultra will help Flipkart’s business associates in offering a development platform for merging their applications within Flipkart application and provide a variety of services to the customers. It will help Flipkart customers to get access to a single shopping destination. It is an idea designed to pave new ways for users to interact with any external application within Flipkart with security and trust. This can help Flipkart business partners (such as, MakeMyTrip, Phone Pe and so on) in getting closer to one of the largest user base in the country by opening their services like booking tickets, bill payments and others for the users. It will further help Flipkart present an ultimate shopping experience to its customers by launching convenience services such as online ticketing, online payment of bills, so on and by including more business categories indirectly into its domain.
```

Comment:
Possible to separate into "What's in it for partners" and "What's in it for users" sub-sections? Shuttling between the two in the same section is a bit confusing to read.